src/xdocs/config_javadoc.xml

```diff
@@ -814,6 +736,47 @@ public int checkReturnTag(final int aTagIndex,
   &lt;property name="excludeScope" value="protected"/&gt;
 &lt;/module&gt;
         </source>
+
+        <p>
+          To configure the check to validate <code>throws</code> tags, you can use following config.
+          ATTENTION: checkstyle do not have information about hierarchy of exception types so usage
```

Comment:
`checkstyle` => `Checkstyle`.
`do not` => `does not`

Reply:
done